drs_filer/errors/errors.py

```diff
@@ -0,0 +1,97 @@
+"""Custom errors, error handler functions and function to register error
```

Comment:
I guess you can use the new FOCA errors here and only overwrite `404` accordingly. Note that there are _two_ different `404` errors in the DRS specs (different descriptions). So you can use and register FOCA defaults first, then override the other two handlers here.